test/input.txt

```diff
@@ -23,7 +23,7 @@
   list
 
 ##########################################################
-# test add person command, setup state for futuer tests
+# test add person command, setup state for future tests
```

Comment:
Good job noticing these small errors, also in the execute method of Command.java